### PR TITLE
Translate Country ranking to Swedish

### PR DESCRIPTION
**Pull request by magol**
Swedish: Translate ranking

### CODE REVIEW

**Comment by magol** [comment]
@ThierryOllivero Thanks, I have fixed this now :-)

**Comment by ThierryOllivero** [comment]
Changes are OK but I don't understand why you changed also the indent size for all json files?

**Comment by magol** [comment]
The default number of spaces is set to 2 in editconfig. But in all json files, the numer of spaces are 4. This made visual studio to automatic reformat each line that i was edit, that was enoying.

**Comment by magol** [comment]
An alternative is to change all translation files to use 2 spaces as indent. Everything else in the repository is using 2 spaces, but the I have to change all lines in all translation files.

File: .editorconfig [hunk 1]
```diff
@@ -5,3 +5,6 @@ indent_style = space
 indent_size = 2
 end_of_line = lf
 insert_final_newline = true
+
+[*.json]
+indent_size = 4
```
Comment (corradio) [inline .editorconfig]: 👍

**Pull request merged**